controllers/rollingupgrade_controller.go

```diff
@@ -237,10 +240,12 @@ func (r *RollingUpgradeReconciler) CallKubectlDrain(ctx context.Context, nodeNam
 
 	// kops behavior implements the same behavior by using these flags when draining nodes
 	// https://github.com/kubernetes/kops/blob/7a629c77431dda02d02aadf00beb0bed87518cbf/pkg/instancegroups/instancegroups.go lines 337-340
-	out, err := runScript(kubeCtlCall+" drain "+nodeName+" --ignore-daemonsets=true --delete-local-data=true --force --grace-period=-1", false, ruObj.Name)
+	script := kubeCtlCall + " drain " + nodeName + " --ignore-daemonsets=true --delete-local-data=true --force --grace-period=-1"
```

Comment:
While were changing this, can you use `fmt.Sprintf` instead?

Reply:
Sure.